# Publications

<p style="text-align: center;">
<a href=https://pubmed.ncbi.nlm.nih.gov/?term=Mathy+CJP%5BAuthor%5D&sort=date>Pubmed</a>
|
<a href=https://scholar.google.com/citations?user=DuBa5oYAAAAJ&hl=en>Google Scholar</a>
</p>

## Pre-print

__Mathy, C. J. P.__, Mishra, P., Flynn, J. M., Perica, T., Mavor, D., Bolon, D. N. A.\*, & Kortemme, T.\* (2022). A complete allosteric map of a GTPase switch in its native network. In __*bioRxiv*__ (p. 2022.04.13.488230). \* denotes co-corresponding authors
- [bioRxiv link](https://doi.org/10.1101/2022.04.13.488230)

```{image} ../images/dms_paper_dark.png
:width: 650px
:align: center
:class: only-dark
:alt: Deep mutational scanning of a GTPase uncovers novel allosteric sites
```

```{image} ../images/dms_paper_light.png
:width: 650px
:align: center
:class: only-light
:alt: Deep mutational scanning of a GTPase uncovers novel allosteric sites
```

## Peer-reviewed publications

Perica, T.\*, __Mathy, C. J. P.\*__, Xu, J., Jang, G. Μ., Zhang, Y., Kaake, R., Ollikainen, N., Braberg, H., Swaney, D. L., Lambright, D. G., Kelly, M. J. S., Krogan, N. J., & Kortemme, T. (2021). Systems-level effects of allosteric perturbations to a model molecular switch. __*Nature*__, 1–6. \* denotes equal contribution
    
- [Journal link](https://doi.org/10.1038/s41586-021-03982-6) | [Pubmed link](https://pubmed.ncbi.nlm.nih.gov/34646016/) | {download}`Paper PDF <../documents/Perica_Mathy_2021.pdf>` | {download}`Supplement PDF <../documents/Perica_Mathy_2021_supp.pdf>`
- Selected for a [Spotlight](https://doi.org/10.1016/j.tibs.2022.01.006) in __*Trends in Biochemical Sciences*__
- Featured in an [article](https://www.ucsf.edu/news/2021/10/421646/how-cells-multitask-magic-molecular-switches) by UCSF

```{image} ../images/gsp1_paper_2021_dark.png
:width: 650px
:align: center
:class: only-dark
:alt: Profiling the cellular effects of interface mutations in the GTPase Ran/Gsp1
```

```{image} ../images/gsp1_paper_2021_light.png
:width: 650px
:align: center
:class: only-light
:alt: Profiling the cellular effects of interface mutations in the GTPase Ran/Gsp1
```

------------------------------------------------------------
Baker, J. J., __Mathy, C. J. P.__, & Schaletzky, J. (2021). A proposed workflow for proactive virus surveillance and prediction of variants for vaccine design. __*PLoS Computational Biology*__, 17(12), e1009624.
- [Journal link](https://doi.org/10.1371/journal.pcbi.1009624) | [Pubmed link](https://pubmed.ncbi.nlm.nih.gov/34914686/) | {download}`Paper PDF <../documents/Baker_2021.pdf>`

------------------------------------------------------------
Bouhaddou, M., Memon, D., Meyer, B., White, K. M., Rezelj, V. V., Correa Marrero, M., Polacco, B. J., Melnyk, J. E., Ulferts, S., Kaake, R. M., Batra, J., …, __Mathy, C. J. P.__, … Krogan, N. J. (2020). The Global Phosphorylation Landscape of SARS-CoV-2 Infection. __*Cell*__. https://doi.org/10.1016/j.cell.2020.06.034
- [Journal link](https://doi.org/10.1016/j.cell.2020.06.034) | [Pubmed link](https://pubmed.ncbi.nlm.nih.gov/32645325/) | {download}`Paper PDF <../documents/Bouhaddou_2020.pdf>`

------------------------------------------------------------
Gordon, D. E., Jang, G. M., Bouhaddou, M., Xu, J., Obernier, K., White, K. M., O’Meara, M. J., Rezelj, …, __Mathy, C. J. P.__, … Krogan, N. J. (2020). A SARS-CoV-2 protein interaction map reveals targets for drug repurposing. __*Nature*__. https://doi.org/10.1038/s41586-020-2286-9
- [Journal link](https://doi.org/10.1038/s41586-020-2286-9) | [Pubmed link](https://pubmed.ncbi.nlm.nih.gov/32353859/) | {download}`Paper PDF <../documents/Gordon_2020.pdf>` | {download}`Supplement PDF <../documents/Gordon_2020_supp.pdf>`
- Our collaborators in the Krogan lab mapped the interactions between human and SARS-COV-2 proteins, and looked to our group for structural interpretation. I discovered a conserved interaction motif in the C-terminal end of Orf6 and modeled it in Rosetta (see figure, purple) using a previously solved crystal structure of the Nup98-Rae1 complex bound to a protein from an unrelated virus, VSV ([Quan et al 2014](https://www.pnas.org/doi/10.1073/pnas.1409076111)). The analysis predicted a methionine was key to the interaction, and mutations to this methionine were later shown to disrupt Orf6 binding to the complex ([Miorin et al 2020](https://www.pnas.org/doi/10.1073/pnas.2016650117)).

```{image} ../images/orf6_dark.png
:width: 650px
:align: center
:class: only-dark
:alt: Profiling the cellular effects of interface mutations in the GTPase Ran/Gsp1
```

```{image} ../images/orf6_light.png
:width: 650px
:align: center
:class: only-light
:alt: Profiling the cellular effects of interface mutations in the GTPase Ran/Gsp1
```

## Other work

My previous work on growth factor engineering formed the basis of an undergraduate honors thesis:

__Mathy, C. J. P.__ (2016). Engineering the NK1 Fragment of the Human Hepatocyte Growth Factor for Dual Use as a Potent Agonist and a Gene Therapy Delivery Vehicle. Stanford Digital Repository. http://purl.stanford.edu/kw533qg7356 {download}`Thesis PDF <../documents/Mathy_HonorsThesis_BIOE.pdf>`